### ISSUE TITLE
Selenium version for better reliabilty

### ISSUE THREAD

**Issue by ading2210** [ading2210/poe-api#195]
With the current Python-only implementation, it's possible for Poe to slightly alter their website which would completely break the library.

Instead, if we switched to a webdriver, minor changes in the client JS would have little practical effect. The largest drawback to this approach, would be the high amount of overhead it would introduce. However, I believe this trade off is worth it if it allows this library to be much more stable.

**Comment by Anandkrishna34** [comment]
idk. I think ok. But i dont understand what you mean. Are you switching to js? Then a big no from me

**Comment by ading2210** [comment]
> But i dont understand what you mean. Are you switching to js?
@Anandkrishna34 

No, my plan is to use browser automation instead of using something requests-based, so the Python code would be responsible for controlling browser actions rather than directly sending the requests.

**Comment by claell** [comment]
The funny thing is that I thought "okay, I'll use this API, as it probably is more stable than the ChatGPT Python packages relying on Selenium". Usually, APIs can be more stable and reliable, but of course not when the company is actively breaking things.

So maybe it's the right way to go. Possibly also some hybrid approach where one authenticates once through Selenium and then the values are stored locally for usage with the API.

**Comment by canxin121** [comment]
i recommend use playright instead, and actually i had made a playright version, but it's abandoned  because the consumption performance is too high.

**Comment by akl7777777** [comment]
> 

You can perform operations such as retrieving form key using a headless browser, instead of implementing all APIs using a regular browser.

**Comment by canxin121** [comment]
> > 
> 
> You can perform operations such as retrieving form key using a headless browser, instead of implementing all APIs using a regular browser.

you can't, now what ever you do,  you need to retrieve tag_id, and retrieving tag_id is as hard as retrieving formkey, and it's different in every operation, so it also need headless browser.

**Comment by GPT5free** [comment]
we will be back，poe！
